engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/videoSettings/VideoSettingsScreen.java

```diff
@@ -229,6 +229,7 @@ public void set(Float value) {
         WidgetUtil.tryBindCheckbox(this, "volumetricFog", BindHelper.bindBeanProperty("volumetricFog", config.getRendering(), Boolean.TYPE));
         WidgetUtil.tryBindCheckbox(this, "inscattering", BindHelper.bindBeanProperty("inscattering", config.getRendering(), Boolean.TYPE));
         WidgetUtil.tryBindCheckbox(this, "cloudShadow", BindHelper.bindBeanProperty("cloudShadows", config.getRendering(), Boolean.TYPE));
+        WidgetUtil.tryBindCheckbox(this, "extraLighting", BindHelper.bindBeanProperty("normalMapping", config.getRendering(), Boolean.TYPE));
```

Comment:
Would it be better with two checkboxes then? We used to have "Environment Effects" which was weirdly vague and covered two separate items that ultimately made more sense as individual checkboxes.

A really fancy and way overblown approach would be to start the config UI in a "Basic" mode with generic/vague option names for fewer grouped option, with an "Advanced" button that would then split out the exact details into individual checkboxes. But that's most likely waaay too much effort for its worth. Just nice and geeky.